git-workflows/bin/git-workflow.sh

```diff
@@ -54,146 +54,195 @@ Function:
 EOF
 }
 
+####################### helper ###################
+formatOutput() {
+  while IFS= read -r line; do
+    log "$line"
+  done
+}
+
+log() {
+  printf "%s [git-workflows] %s\n" "$(date '+%Y-%m-%d %T')" "$1"
+}
+
+changedirOrExit() {
+  TARGET=$1
+  log "switch to dir '$TARGET'"
+  cd "$TARGET"
+  ERR=$?
```

Comment:
die Stelle kannst du glaub ich nicht erreichen durch das "set -e" in der Main Methode.